test/unit/oracle-aggregator.spec.ts

```diff
@@ -176,37 +176,36 @@ describe('OracleAggregator', () => {
   });
 
   describe('addOrModifySupportForPair', () => {
```

Comment:
new case is not being tested, right? I mean, theoretically is the first case. Might be worth to put it on an e2e, probably.

Reply:
It's being tested in `when('oracle was forced but it lost support', () => { ... `